local-dev.yml

```diff
@@ -105,25 +113,27 @@ services:
     ports:
       - "8983"
     command: ["solr-foreground"]
-  nginx:
-    build:
-      context: ./nginx
-      dockerfile: Dockerfile-nginx
-    hostname: nginx
-    container_name: nginx
-    links:
-      - hydroshare:hydroshare
-    depends_on:
-      - hydroshare
-    ports:
-      - "80:80"
-    volumes:
-      - "./log/nginx:/var/log/nginx"
-      - ".:/hydroshare"
-      - "temp_vol:/hs_tmp"
-      - "share_vol:/irods/cache"
-      - "idata_vault_vol:/irods/data"
-      - "iuser_vault_vol:/irods/user"
+  # TODO: nginx container does not currently work
```

Comment:
nginx has not been working on local dev for a long time. I removed it here because it takes time in the build process.